packages/dx-react-core/docs/guides/fundamentals.md

```diff
@@ -7,15 +7,15 @@ React Core provides components for creating a plugin-based component.
 A plugin-based component should adhere to the following principles:
 
 ##### [Inversion of Control](https://en.wikipedia.org/wiki/Inversion_of_control){:target="_blank"}
- 
+
 A plugin shares its items (Getters, Actions, Templates) within an IoC (Inversion of Control) container and can use the ones other plugins share.
- 
+
 A plugin based component consists of the following child components:
 
-- Plugin Host component  
```

Comment:
We should keep double whitespaces because in this case markdown is broken. You can define the `trim_trailing_whitespace` property to `false` in the `.editorconfig` file.

Reply:
I've updated a list item format.